setup.py

```diff
@@ -10,10 +10,7 @@
     'azure-cli',
     'jmespath',
     'prompt_toolkit',
-    'pylint',
     'pyyaml',
-    'autopep8',
-    'flake8',
```

Comment:
Doesn't look like this change should be part of this PR.
The change is already in master.